### ISSUE TITLE
the word math dose not exist cant fix it

### ISSUE THREAD

**Issue by RaDzez** [DaniDevy/FPS_Movement_Rigidbody#3]

**Comment by Rockstar2K** [comment]
only change all the math´s with Mathf

**Comment by KoniChristochaou** [comment]
it works duh unless you didn't see Dani's tutorial video

**Comment by Adham4678** [comment]
No, all of these other comments are incorrect. Firstly, changing Math to Mathf will only convert int to float. The solution is to use: `using System;` at the top. Math is a function included in the System  namespace.

**Comment by bakane6030** [comment]
> No, all of these other comments are incorrect. Firstly, changing Math to Mathf will only convert int to float. The solution is to use: `using System;` at the top. Math is a function included in the System  namespace.

Just to let you know Mathf is a struct made by Unity and included in the UnityEngine namespace.

**Comment by SomeDudeWhoExists** [comment]
I think that is a prob with your thing, not the term because `math` exists.
